src/App.js

```diff
@@ -1,13 +1,42 @@
 import React from 'react';
-
+import { GoodsList } from './components/GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export class App extends React.Component {
+  state = {
+    goods: [],
+  };
+
+  setGoods = (goods) => {
+    this.setState({
+      goods,
+    });
+  };
```

Comment:
I think it's better to use setState inside of showGoods func, no need to create a separate method for it